readme.md

```diff
@@ -1,3 +1,42 @@
 # Product Owner Guide
 
 A rough guide for those who are product owners on a dwyl project.
+
+## What is a Product Owner?
+
+**A Product Owner (PO)** is someone who represents the views and needs of the
+target audience, stakeholders and the business they represent in agile scrum
+methodology. They must understand and communicate the vision of the product to
+the rest of the team.
+
+They are the liaison between customers and the development team. This means that
+they are responsible for working with stakeholders and user groups to determine
+their needs. Then through collaboration with the Scrum Master and developer team
+they prioritise and discuss these ideas moulding them into features to be built.
+
+**PO Responsibilities:**
+1. Owner and creator of the product backlog* :crown:
```

Comment:
We're already getting into terminology that new POs (our audience) won't understand (like 'Scrum master' on line 14 above and 'product backlog' here), which will make people switch off pretty early on. Any thoughts on how we can mitigate this?

Reply:
Ooooooooh, I see what you did here with the asterisk 👍 

Still a question for other things like `scrum master`, `user testing` and `UI`.

Not a blocker for this PR, but worth an issue or solution.

Reply:
I've created #22 for this, nice suggestion @iteles